### ISSUE TITLE
How to save the binarized model?

### ISSUE THREAD

**Issue by iamweiweishi** [zhaoweicai/hwgq#6]
According to my understanding, the weights of the trained model were not binarized directly. 
So, do I need to binarize the trained model with parameter 'scale' and then re-save the model?

**Comment by zhaoweicai** [comment]
Yes, and we have also provided the binarized models, please check README.

**Comment by iamweiweishi** [comment]
Hi, I trained one model using res50. And I tried to convert the trained models to binarized ones. Unfortunately, never succeed. 
I found the parameter 'alphas_' in 'binary_conv_layer.cpp', but I have no idea how to extract the parameter in trained models.
Could you pls send me (nuptsww@126.com) some script example for reference. 
Thanks a lot.
